recipe/conda_build_config.yaml

```diff
@@ -1,8 +1,8 @@
 clang_variant:
   - default
-  # - cling_v0.6       # tag is cling-v0.6
+  - cling_v0.6       # tag is cling-v0.6
```

Comment:
Delete this as we are not even building cling v0.6 anymore

Reply:
Fixed. This was a failed rebase.